build.gradle

```diff
@@ -122,3 +122,17 @@ task installPythonTools(type: Exec) {
     executable 'bash'
     args '-c', condaCommand('pip install conan grpcio-tools twine')
 }
+
+// To use this task the following properites should be set in your gradle.properties:
+//   pypiUser
```

Comment:
Shall we change the properties for consistency to `pypi.user` and `pypi.password`?

Reply:
Ah yeah. Will do.